sources/packages/backend/libs/integrations/src/services/sfas/sfas-individual.service.ts

```diff
@@ -14,8 +17,33 @@ export class SFASIndividualService
   extends DataModelService<SFASIndividual>
   implements SFASDataImporter
 {
+  private readonly bulkUpdateStudentIdSQL: string;
+  private readonly bulkUpdateBCSLDisbursementOverawardSQL: string;
+  private readonly bulkUpdateCSLFDisbursementOverawardSQL: string;
+  private readonly bulkInsertBCSLDisbursementOverawardSQL: string;
+  private readonly bulkInsertCSLFDisbursementOverawardSQL: string;
```

Comment:
I know CSLP is a part time overaward and current sfas integration may not have it. 
But looking at the column(in sfas_individuals) it seems to be very generic (csl_overaward). Is there a possibility of part time overaward coming through this integration?

Reply:
No, there is no possibility of part-time coming in the integration because there is no concept of a part-time overaward on SAIL. Part-time overaward exists only on SIMS.

Reply:
As far as I know, that is not expected to happen for now.